build.sbt

```diff
@@ -237,7 +237,7 @@ lazy val testShared = (project in file("modules/test-shared"))
   )
 
 lazy val sdk = (project in file("modules/sdk"))
-  .dependsOn(shared % "compile->compile;test->test", testShared % Test, keytool, kernel)
+  .dependsOn(shared % "compile->compile;test->test", testShared % Test, keytool, kernel, dagShared)
```

Comment:
As a general thought, we probably need to rethink how to organize modules. W have to many modules I believe, without having a clear separation/responsibility.
`kernel`, `shared`, `test-shared`, `dag-shared`, `sdk` and also `keytool` (which is also used as a library).

Reply:
I'd reorganize them as part of the Tessellation SDK work from the roadmap

Reply:
I agree. It would be best to do it before or at least during sdk development.